Request/ParamFetcher.php

```diff
@@ -52,6 +52,10 @@ class ParamFetcher implements ParamFetcherInterface
      */
     public function __construct(ContainerInterface $container, ParamReaderInterface $paramReader, RequestStack $requestStack, ValidatorInterface $validator = null)
     {
+        if (null === $validator) {
+            @trigger_error("Using no validator is deprecated since version 2.6. It won't be possible anymore in 3.0.");
```

Comment:
We should reword this a bit like this:

```php
@trigger_error(sprintf('Using no validator is deprecated since FOSRestBundle 2.6. The `$validator` constructor argument of the `%s` will become mandatory in 3.0.', __CLASS__), E_USER_DEPRECATED);
```

Reply:
My bad, I forgot E_USER_DEPRECATED type, sorry for that :/